meta-resin-common/recipes-support/resin-vars/resin-vars/config-json.path

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Config.json path watch
+Requires=mnt-boot.mount
+After=mnt-boot.mount
+
+[Path]
+PathChanged=/mnt/boot/config.json
+
+[Install]
+WantedBy=mnt-boot.mount
```

Comment:
Ok, I read back up on this and the reasoning is because this watcher needs to be readded whenever /mnt/boot is mounted - because it looks into that mount point, and if that mount point is remounted it needs to be triggered on the new/remounted mount